test/e2e/runner/main.go

```diff
@@ -189,17 +210,24 @@ func NewCLI() *CLI {
 	})
 
 	cli.root.AddCommand(&cobra.Command{
-		Use:   "logs",
-		Short: "Shows the testnet logs",
+		Use:     "logs [node (optional)]",
+		Short:   "Shows the testnet or a specefic node's logs",
+		Example: "runner logs valiator03",
 		RunE: func(cmd *cobra.Command, args []string) error {
+			if len(args) == 1 {
+				return execComposeVerbose(cli.testnet.Dir, "logs", args[0])
+			}
 			return execComposeVerbose(cli.testnet.Dir, "logs")
 		},
 	})
 
 	cli.root.AddCommand(&cobra.Command{
-		Use:   "tail",
+		Use:   "tail [node (optional)]",
 		Short: "Tails the testnet logs",
 		RunE: func(cmd *cobra.Command, args []string) error {
+			if len(args) == 1 {
```

Comment:
This should error if `len(args) > 1`. Or is that validated elsewhere?

Reply:
I had thought just to ignore it. Do you think it's better to return an error? I can also set max arguments which should return an error if it is exceeded

Reply:
Yeah, return an error -- otherwise people will wonder why stuff isn't working.